content/en/docs/concepts/overview/_index.md

```diff
@@ -1,7 +1,97 @@
 ---
+reviewers:
+- bgrant0607
+- mikedanese
 title: "Overview"
+description: >
+  Kubernetes is a portable, extensible, open source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available.
+content_type: concept
 weight: 20
-description: Get a high-level outline of Kubernetes and the components it is built from.
+card:
+  name: concepts
+  weight: 10
 sitemap:
   priority: 0.9
```

Comment:
After the move, should this 0.9 priority be updated?

Reply:
Ah, yes, fixed this as well